backend/app/src/main/kotlin/io/tolgee/api/v2/hateoas/user_account/UserAccountModel.kt

```diff
@@ -11,6 +11,8 @@ data class UserAccountModel(
   val username: String,
   var name: String?,
   var emailAwaitingVerification: String?,
+  var mfaEnabled: Boolean,
```

Comment:
This model is used also publicaly. Pease create separate model. We don't want to provide this information to everybody.